packages/importer/src/index.ts

```diff
@@ -135,23 +109,36 @@ class OneSchemaImporter extends EventEmitter {
     parent.append(this.iframe)
   }
 
-  launch(config: OneSchemaImporterConfig) {
+  /**
+   * Launch will show the OneSchema window and initialize the importer session
+   * @param launchConfig optionally pass in config overrides or values not passed into constructor
+   */
+  launch(launchConfig?: Partial<OneSchemaLaunchConfig>) {
     window.addEventListener("message", this.#eventListener)
     this.#show()
 
-    const { options, ...rest } = config
+    const mergedConfig = merge({}, this.#config, launchConfig)
+    const message: any = { messageType: "init" }
+    message.options = mergedConfig.config
+
+    message.userJwt = mergedConfig.userJwt
+    if (!message.userJwt) {
+      console.error("OneSchema config error: missing userJwt")
+      return
+    }
+
+    message.templateKey = mergedConfig.templateKey
+    if (!message.templateKey) {
+      console.error("OneSchema config error: missing templateKey")
+      return
+    }
```

Comment:
I could let the import session fail instead, in that way there would probably be a UI change in dev mode. I could also console log and not return so maybe both would happen.